src/main.scss

```diff
@@ -29,19 +24,14 @@
 @import 'line-heights';
 @import 'links';
 @import 'margins';
-<<<<<<< HEAD
 @import 'object-fits';
 @import 'opacity';
 @import 'overflows';
 @import 'paddings';
 @import 'positions';
 @import 'text';
+@import 'top-right-bottom-left';
```

Comment:
Perhaps we call this file `coordinates` or `directions`? Another options would be to add them to the `_positions.scss` since these classes only work on positioned elements anyway; https://developer.mozilla.org/en-US/docs/Web/CSS/top.

Reply:
positions dot sass !!!!!